### PR TITLE
feat: associate commits to sentry

### PR DESCRIPTION
**Pull request by TheDome**

### CODE REVIEW

File: .github/workflows/release-docker-image.yaml [hunk 1]
```diff
@@ -49,6 +49,16 @@ on:
...
 
+      sentry-auth-token:
+        description: Sentry authentication token
+        required: false
+      sentry-org:
+        description: Sentry organization
+        required: false
+      sentry-project:
+        description: Sentry project
+        required: false
```
Comment (RappC) [inline .github/workflows/release-docker-image.yaml:60]: Why are these secrets? 
Additionally I would add default values for both. 
- org: "neohelden"
-  project: <github-repo-name>

**Review by RappC — CHANGES_REQUESTED** [review]

File: .github/workflows/release-docker-image.yaml (hunk 1 above)
Comment (TheDome) [reply .github/workflows/release-docker-image.yaml:60]: Sentry proposes those as secrets...

Comment (RappC) [reply .github/workflows/release-docker-image.yaml]: I think this is a copy&paste error in the sentry readme. It does not make sense, to hide these details in a secret.

File: .github/workflows/release-docker-image.yaml [hunk 2]
```diff
@@ -30,6 +30,15 @@ on:
         default: kustomizations/overlays/release
         required: false
 
+      sentry-org:
+        description: Sentry organization
+        required: false
```
Comment (RappC) [inline .github/workflows/release-docker-image.yaml:35]: what about a default value `neohelden`?

File: .github/workflows/release-docker-image.yaml [hunk 3]
```diff
@@ -30,6 +30,15 @@ on:
...
+        type: string
+      sentry-project:
```
Comment (RappC) [inline .github/workflows/release-docker-image.yaml:37]: What about a default `github repo name`?

File: .github/workflows/release-docker-image.yaml [hunk 4]
```diff
@@ -103,6 +116,25 @@ jobs:
               }'            
           fi
 
+  associate-commits:
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+    if: ${{ secrets.sentry-auth-token }} != ''
```
Comment (RappC) [inline .github/workflows/release-docker-image.yaml:122]: I think brackets are not needed here.

```suggestion
    if: secrets.sentry-auth-token != ''
```

**Review by RappC — CHANGES_REQUESTED** [review]
never mind

**Comment by RappC** [comment]
> Shouldn't we move this into the composite action `release-docker-image`?

Never mind. I just saw that we deprecated that composite.

**Review by RappC — APPROVED** [review]

**Pull request merged**